src/frontend/src/common/utils/featureFlags.tsx

```diff
@@ -0,0 +1,24 @@
+import { hasQueryParam } from ".";
```

Comment:
Can you explain how this works to me? I've never seen this usage for importing from something other than an `index.js` file. Is it effectively searching all the exports in the directory for that named export? I had been under the impression that you always had to specify the file being imported from OR specify a directory and it would default to importing from `index` in that directory. But... obviously I'm wrong. So what's the deal with `"."`?

Reply:
Oops, this was an autocompleted line. But the answer here is that there is an index.tsx file in the same directory that imports all the util files and re-exports them. That makes each function available from the directory level (like shown here), or from the file level (as if I were going to do `... from "./urlUtils";`

Reply:
Ah, thank you!